### PR TITLE
Update math_util.c

### PR DESCRIPTION
**Pull request by KazeEmanuar**
yipped backface culling

### CODE REVIEW

**Comment by KazeEmanuar** [comment]
yip it to the net!

File: src/engine/math_util.c [hunk 1]
```diff
@@ -1340,7 +1340,7 @@ s32 ray_surface_intersect(Vec3f orig, Vec3f dir, f32 dir_length, struct Surface
     // Determine the cos(angle) difference between ray and surface normals.
     f32 det = vec3f_dot(e1, h);
     // Check if we're perpendicular from the surface.
```
Comment (thecozies) [inline src/engine/math_util.c:1342]:
```suggestion
    // Check if we're perpendicular or pointing away from the surface.
```

File: src/engine/math_util.c [hunk 2]
```diff
@@ -1340,7 +1340,7 @@ s32 ray_surface_intersect(Vec3f orig, Vec3f dir, f32 dir_length, struct Surface
...
-    if ((det > -NEAR_ZERO) && (det < NEAR_ZERO)) return FALSE;
+    if ((det < NEAR_ZERO)) return FALSE;
```
Comment (thecozies) [inline src/engine/math_util.c:1343]: too many parenthesis
```suggestion
    if (det < NEAR_ZERO) return FALSE;
```

**Review by thecozies — CHANGES_REQUESTED** [review]
close!

**Review by Reonu — APPROVED** [review]
gogogo

**Comment by NyakoFox** [comment]
yip

**Review by thecozies — APPROVED** [review]
yip

**Comment by KazeEmanuar** [comment]
SHUT UP

Am Mi., 8. Juni 2022 um 21:10 Uhr schrieb thecozies <
***@***.***>:

**Comment by axollyon** [comment]
hi kayze

**Comment by Reonu** [comment]
yo kaze I like your inbox

**Review by Mr-Wiseguy — APPROVED** [review]

**Pull request merged**